src/util/responses.js

```diff
@@ -1,14 +1,14 @@
 module.exports = {
   "en-US": {
     "NO_RESULTS_FOUND": "{0} did not return any results on YouTube.",
-    "ASK_TO_PLAY": "I found a video called {0}. Would you like me to play it?",
+    "ASK_TO_PLAY": "I found a video called {0}. Would you like me to download it now?",
```

Comment:
Changed "play" to "download" since there could be a long wait until the play finally starts. Also added "now" to the end to make "it" sound better 😃